infra.bs

```diff
@@ -1424,6 +1425,23 @@ ascending order, with |a| being less than |b| if |a|'s second <a for=struct>item
 <a>code unit less than</a> |b|'s second <a for=struct>item</a>, gives the result « (404,
 "<code>Not Found</code>"), (200, "<code>OK</code>"), (null, "<code>OK</code>") ».</p>
 
+<div algorithm>
+    <dfn>Sequence iteration</dfn> over an ordered sequence |seq|,
```

Comment:
"ordered sequence" is not defined. So I think the desire to share code here between lists and maps doesn't work well. You could assemble a list of map entries and operate on them, but doing so is a lot of work...

I think it might be best to just define it separately, with some duplication.

Reply:
Yeah, there's always a fine line between tautological definitions and real definitions in Infra, since we haven't decided to ground everything in first-order logic. For this case I think we're trying to be precise about a specific algorithm so I think we should try to be explicit.